app/(default)/page.tsx

```diff
@@ -1,11 +1,15 @@
+import Image from 'next/image';
 import Link from 'next/link';
 import { ReactElement } from 'react';
 
+import logo from './assets/nilshindermann_logo.svg';
```

Comment:
@llettieri is there a better way to import an svg logo?

Reply:
In my projects I use a custom component for SVG images. Feel free to take a look in one of my projects.